README.md

```diff
@@ -81,6 +81,6 @@ That said, if you have questions, reach out to us
 [Scalable Microservices with Kubernetes]: https://www.udacity.com/course/scalable-microservices-with-kubernetes--ud615
 [Submit Queue]: http://submit-queue.k8s.io/#/e2e
 [Submit Queue Widget]: http://submit-queue.k8s.io/health.svg?v=1
-[troubleshooting guide]: http://kubernetes.io/docs/troubleshooting
+[troubleshooting guide]: https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/ 
```

Comment:
Yeah, according to https://github.com/kubernetes/kubernetes.github.io/pull/3337 , this change is right.